src/main/java/com/siftscience/VerificationCheckRequest.java

```diff
@@ -0,0 +1,44 @@
+package com.siftscience;
+
+import com.siftscience.model.VerificationCheckFieldSet;
+import okhttp3.OkHttpClient;
+import okhttp3.HttpUrl;
+import okhttp3.Response;
+import okhttp3.Request;
+import okhttp3.Credentials;
+import okhttp3.MediaType;
+import okhttp3.RequestBody;
+
+import java.io.IOException;
+
+
+/*
```

Comment:
Should this be a document style comment?
```suggestion
/**
```